File: 9. Palindrome Number.js
//9. Palindrome Number



/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    let rev = 0;
    let temp = x;
    while(x>0){
        rev = (rev*10)+x%10;
        x=Math.floor(x/10);
    }
    return rev===temp;
};

File: 9. Palindrome Number.java
//9. Palindrome Number


class Solution {
    public boolean isPalindrome(int num) {
        int rev = 0;
        int temp=num;
		while(num>0) {
			rev=(rev*10)+num%10;
            num/=10;
		}
		return rev == temp;
    }
}
